otelcollector/prometheusreceiver/targetallocator/manager.go

```diff
@@ -172,20 +178,16 @@ func (m *Manager) applyCfg() error {
 		return err
 	}
 
-	if err := m.webHandler.ApplyConfig(m.promCfg); err != nil {
-		return err
-	}
```

Comment:
Thanks Rashmi, could you add back in the webhandler code for this file? This was new with the last collector upgrade where this file was refactored out. We can add this step to the upgrades doc too

Reply:
Thanks Grace, just added and updated our docs too